src/feature.js

```diff
@@ -31,7 +31,7 @@ var feature = function (arg) {
 
   var m_this = this,
       s_exit = this._exit,
-      m_selectionAPI = arg.selectionAPI === undefined ? false : arg.selectionAPI,
+      m_selectionAPI = arg.selectionAPI === undefined ? false : !!arg.selectionAPI,
```

Comment:
We could use `Boolean()` instead.  `!!` is substantially faster than `Boolean()` in Chrome, though, and, it is a common enough javascript idiom, that I think it is fine.